src/intentLogic/yes.js

```diff
@@ -11,6 +11,7 @@ module.exports = function () {
     switch (attributes.lastIntent) {
         case 'GetPodcastIntent':
             const podcastDirective = helpers.getPodcastDirective(attributes.podcastUrl, attributes.podcastTitle);
+            console.log("Playing podcast " + attributes.podcastUrl);
```

Comment:
Yes - this is how we log podcasts at the moment. We're planning to do a proper Ophan integration of all podcasts in the near future.